AU2/plugins/custom_plugins/PolicePlugin.py

```diff
@@ -88,122 +93,205 @@ def __init__(self):
                 "Police -> Set Ranks",
                 self.ask_set_ranks,
                 self.answer_set_ranks
+            ),
+            ConfigExport(
+                "PolicePlugin_toggle_auto",
```

Comment:
Not crucial, but as you mentioned you're concerned about config bloat, you could probably merge these into one config option with a [`SelectorList`](https://github.com/jsyiek/AU2/blob/master/AU2/html_components/SelectorList.py) (all you're doing is toggling on/off, so selection with the list = true, and not = false).